pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -169,6 +169,23 @@ func (k *KogitoServiceSpec) AddEnvironmentVariable(name, value string) {
 	return
 }
 
+// AddEnvironmentVariableFromSecret adds a new environment variable from the secret under the key
+func (k *KogitoServiceSpec) AddEnvironmentVariableFromSecret(name, secret, key string) {
```

Comment:
Meaning of those parameters is not clear on the first look. I would suggest to adjust them like this:
```suggestion
func (k *KogitoServiceSpec) AddEnvironmentVariableFromSecret(name, secretName, secretKey string) {
```

Reply:
I see a difference between secret and secretName but key and secretKey makes no difference for me in this context. But sure, will change it :)

Reply:
Well, when I read the `key` for the first time I didn't associated it with key in secret, but was thinking about some key in environment variable.

Reply:
I see now. I have also changed `name` to `variableName` because when I looked at it after some time, it was not enough. Thanks for the input!